xAPI.md

```diff
@@ -2948,6 +2948,13 @@ do not match.
 
 * The LRS MAY respond before Statements that have been stored are available for retrieval.
 
+###### Activity Provider Requirements
+
+* Activity Providers SHOULD use POST instead of PUT. 
+* When PUTing statements, the "id" property of the Statement SHOULD* be used. 
```

Comment:
I don't see much value of SHOULDstarring this, but I'm not especially invested in it either.